dev/com.ibm.ws.threading/src/com/ibm/ws/threading/internal/PolicyExecutorImpl.java

```diff
@@ -959,4 +960,43 @@ public void shutdown() {
         enqueue(policyTaskFuture, maxWaitForEnqueueNS.get(), null);
         return policyTaskFuture;
     }
+
+    public void introspect(PrintWriter out) {
+        final String INDENT = "  ";
+        final String DOUBLEINDENT = INDENT + INDENT;
+        out.println(identifier);
+        out.println(INDENT + "coreConcurrency = " + coreConcurrency);
+        out.println(INDENT + "maxConcurrency = " + maxConcurrency);
+        out.println(INDENT + "maxQueueSize = " + maxQueueSize);
+        out.println(INDENT + "maxWaitForEnqueue = " + maxWaitForEnqueueNS.get() + " ms");
```

Comment:
Inconsistent unit of measure.  Field is nanoseconds, but is displayed as milliseconds (ms).
Given that the user configures as milliseconds, it will never have granularity beyond milliseconds, so it does make sense to display as milliseconds, however, you need to convert the value to milliseconds before outputting it.